### PR TITLE
NOTIF-465 delete applications in admin UI

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

**Comment by aferd** [comment]
i am fixing the commits above.

**Comment by josejulio** [comment]
Don't forget the screenshots please

File: admin-console/src/main/webapp/src/pages/BundlePage.tsx [hunk 1]
```diff
@@ -114,17 +155,20 @@ export const BundlePage: React.FunctionComponent = () => {
...
                                 </Td>
+                                <Td>{ a.id }</Td>
                                 <Td>
                                     <Button className='edit' type='button' variant='plain'
+                                        onClick={ () => editApplication(a) }
                                     > { <PencilAltIcon /> } </Button></Td>
                                 <Td>
                                     <Button className='delete' type='button' variant='plain'
+                                        onClick={ () => deleteApplicationModal(a) }
                                     >{ <TrashIcon /> } </Button></Td>
```
Comment (josejulio) [inline admin-console/src/main/webapp/src/pages/BundlePage.tsx:172]: I think we could group the actions in a kebab: https://patternfly-react.surge.sh/components/table#composable-actions

**Comment by aferd** [comment]
https://user-images.githubusercontent.com/86322239/152873143-8fb34ee2-7f80-4963-801a-1427b400b146.mov

**Comment by josejulio** [comment]
@aferd There are some lint failures

**Comment by aferd** [comment]
https://user-images.githubusercontent.com/86322239/153485687-c27d1bf2-3fa3-4287-a3df-70037190ca6f.mov

**Review by josejulio — APPROVED** [review]

**Pull request merged**